src/mscorlib/src/System/String.Comparison.cs

```diff
@@ -1144,5 +1144,16 @@ public override int GetHashCode()
 
             return referenceCulture.CompareInfo.IsPrefix(this, value, ignoreCase ? CompareOptions.IgnoreCase : CompareOptions.None);
         }
+
+        [Pure]
+        public Boolean StartsWith(Char value)
```

Comment:
I thought the same, but I was not sure if I should use that syntax, I didn't see much code wrote like that. I'll simplify it